priv/repo/seeds.exs

```diff
@@ -47,14 +46,16 @@ defmodule CsGuide.Import do
   def venues_1(csv) do
     csv
     |> csv_to_map(
-      ~w(nil venue_name nil description venue_types nil nil nil nil nil nil nil address city region country postcode latitude longitude opening_hours phone_number email website twitter facebook instagram)a
+      ~w(nil venue_name nil description venue_types nil nil nil nil nil nil nil address_line_1 city region country postcode latitude longitude opening_hours phone_number email website twitter facebook nil nil nil instagram)a
```

Comment:
As previously discussed, I prefer the **`address`** to be **`address`** rather than **`address_line_1`** for simplicity.